### PR TITLE
Adding AttestationClient source code

### PR DESCRIPTION
**Pull request by Bab95**
Adding source code and build scripts of AttestationClient

### CODE REVIEW

File: client-library/src/Attestation/AttestationClient/lib/DynamicLibrary/debian/changelog [hunk 1]
```diff
@@ -0,0 +1,7 @@
+azguestattestation1 (1.0.2) stable; urgency=medium
+  * Initial release of azguestattestation library.
+
+ -- Vinay Kotak <vikotak@microsoft.com>  Thu, 22 July 2021 17:45:00 -0700
```
Comment (akashgupta29) [inline client-library/src/Attestation/AttestationClient/lib/DynamicLibrary/debian/changelog:4]: are these needed?

File: client-library/src/external/VbsVmCrypto/VbsVmCrypto.h [hunk 2]
```diff
@@ -0,0 +1,399 @@
+/*++
+
+Copyright (c) 2014  Microsoft Corporation
+
+Module Name:
+
+    vbsvmcrypto.h
```
Comment (akashgupta29) [inline client-library/src/external/VbsVmCrypto/VbsVmCrypto.h:7]: Please remove

File: client-library/src/Attestation/AttestationClient/lib/DynamicLibrary/debian/changelog (hunk 1 above)
Comment (Bab95) [reply client-library/src/Attestation/AttestationClient/lib/DynamicLibrary/debian/changelog:4]: changelog file is needed for building debian package

Comment (Bab95) [reply client-library/src/Attestation/AttestationClient/lib/DynamicLibrary/debian/changelog]: removed as suggested

**Comment by Bab95** [comment]
@akashgupta29  addressed comments over PR. Please review it again.

File: client-library/src/docker/DockerFile [hunk 3]
```diff
@@ -0,0 +1,118 @@
+FROM mcr.microsoft.com/mirror/docker/library/ubuntu:18.04
```
Comment (akashgupta29) [inline client-library/src/docker/DockerFile:1]: should we pull the image from some public repo?

Comment (agowdamsft) [reply client-library/src/docker/DockerFile:1]: No, this is recommend as public repo throttles.

**Comment by agowdamsft** [comment]
@akashgupta29 #sign-off from my side.

File: client-library/src/docker/DockerFile (hunk 3 above)
Comment (Bab95) [reply client-library/src/docker/DockerFile:1]: yes this image is msft internal only. I will change it to use public repo ubuntu image.

**Review by akashgupta29 — APPROVED** [review]

**Pull request merged**